### DISCUSSION TITLE
when 4.x is release?

### DISCUSSION THREAD

**Discussion by walfud** [ExpediaGroup/graphql-kotlin#947] [General] [answered]
when 4.x is release?

**Comment by smyrick** [comment, marked as answer]
Hello @walfud,

We have a couple issues we were hoping to still get in the next major release as they are potential breaking changes: https://github.com/ExpediaGroup/graphql-kotlin/milestone/4

That being said, the latest alpha release, [4.0.0-alpha.7](https://github.com/ExpediaGroup/graphql-kotlin/releases/tag/4.0.0-alpha.7), should be production ready. We are just reserving the release to add any additional breaking changes. If you would like to help us get the full release our sooner, contributions are always welcome.

Any bug fixes that we have made to the `master`/`4.x.x` release have also been backported to our current release in the `3.x.x` branch.

**Reply by walfud** [reply]
got it，thanks。
